dockers/docker-base-buster/sources.list

```diff
@@ -6,3 +6,5 @@ deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster mai
 deb [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free
 deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free
 deb [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster-backports main contrib non-free
+deb [arch=amd64] http://packages.trafficmanager.net/debian/debian buster main contrib non-free
+deb [arch=amd64] http://packages.trafficmanager.net/debian/debian buster-updates main contrib non-free
```

Comment:
>buster-updates [](start = 66, length = 14)

Wondering if you need another line for buster-backports?

Reply:
buster-backports added, thanks.